README.md

```diff
@@ -44,8 +44,10 @@ non-production Kubernetes clusters.
     * [Reconciler](#reconciler)
     * [Scale](#scale)
     * [Kube API Retry](#kube-api-retry)
+    * [Informer Cache](#informer-cache)
```

Comment:
This is getting a bit long for the README file. Would it be helpful for the users to have a separate `docs` folder with markdown files for different sections? Similar to [this](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler/docs).

Reply:
Yeah, good call. Could you create an issue for this please? We can implement separately.